rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc

```diff
@@ -67,6 +58,37 @@ A user needs privileges to view and use the RPA Manager modules. This ensures th
 . Repeat the previous step until the user has all needed privileges.
 . Click *Save*.
 
+== Change Process Assignments of a User
+
+Instead of editing a project and adding a user to its project team, change several project assignments of a user at once:
```

Comment:
"change several..." > "you can change several..."

Since it is an either-or situation, this is something they don't have to do. The "you can" reflects that in this context.